buildJDK.sh

```diff
@@ -18,11 +18,14 @@ fi
 pushd ${MANDREL_REPO}/substratevm
 MANDREL_VERSION=${MANDREL_VERSION:-$(git describe)}
 popd
+JAVA_VERSION="$( java --version | sed -e 's/.*build \([^) ]*\)).*/\1/;t;d' | tr +- . )"
```

Comment:
`java --version` should respect `JAVA_HOME`
```suggestion
JAVA_VERSION="$( ${JAVA_HOME}/bin/java --version | sed -e 's/.*build \([^) ]*\)).*/\1/;t;d' | tr +- . )"
```